### PR TITLE
Annotations and format specifiers

### PR DESCRIPTION
**Pull request by neesingh-rh**
Added the annotations and changed the way of using format specifiers
Update: #45

### CODE REVIEW

File: redant_libs/support_libs/io_ops.py [hunk 1]
```diff
@@ -1,91 +1,97 @@
...
-if __name__ == "__main__":
-    R.establish_connection()
-    #volume_mount("10.70.43.228","test-vol","test_dir")
-    create_file_using_touch("test_file")
-    create_dir("test_dir")
-    list_files_on_root_dir()
-   
+class io_ops:
+
+    def volume_mount(self ,node :str ,server :str ,volname :str, dir :str ,force : bool=False):
```
Comment (nik-redhat) [inline redant_libs/support_libs/io_ops.py:3]: What does this do?

Comment (nik-redhat) [reply redant_libs/support_libs/io_ops.py:3]: Do we want this @srijan-sivakumar ?

**Comment by nik-redhat** [comment]
Also, it's a good practice to `sign-off` your commits.

File: redant_libs/support_libs/io_ops.py (hunk 1 above)
Comment (srijan-sivakumar) [reply redant_libs/support_libs/io_ops.py:3]: Are you referring to function annotations ?

Comment (srijan-sivakumar) [reply redant_libs/support_libs/io_ops.py:3]: @nik-redhat ^^

**Review by srijan-sivakumar — APPROVED** [review]

File: redant_libs/support_libs/io_ops.py (hunk 1 above)
Comment (nik-redhat) [reply redant_libs/support_libs/io_ops.py:3]: Yes, what does the `:` do in the argument?

Comment (neesingh-rh) [reply redant_libs/support_libs/io_ops.py:3]: This is the type annotation used in python to let others know about the type of argument passed.

Comment (srijan-sivakumar) [reply redant_libs/support_libs/io_ops.py:3]: Ahh, it follows the same syntax as golang without the actual type checking, it is basically saying node is of type str ( But funny part is that python interpreter still takes in any value you give it :/ ) But if you work on IDE...that's where this comes handy plus anyone reading your code..

**Pull request merged**